Parsers/ASimFileEvent/Parsers/imFileEvent.yaml

```diff
@@ -28,5 +43,6 @@ ParserQuery: |
     vimFileEventMicrosoftSharePoint,
     vimFileEventMicrosoftSysmon,
     vimFileEventAzureTableStorage,
-    vimFileEventMicrosoftWindowsEvents,
-    vimFileEventNative
\ No newline at end of file
+    vimFileEventsMicrosoftWindowsEvent,
```

Comment:
@jayeshprajapaticrest 
an extra 's' has been added here. it should be vimFileEventMicrosoftWindowsEvents

Reply:
Done